src/Util/check_hls_status.php

```diff
@@ -13,6 +13,7 @@ function fetch(string $url): array
     curl_setopt($ch, CURLOPT_SSL_VERIFYPEER, false);
     curl_setopt($ch, CURLOPT_HEADER, true);
     curl_setopt($ch, CURLOPT_FOLLOWLOCATION, true);
+    curl_setopt($ch, CURLOPT_TIMEOUT, 2);
```

Comment:
@okaufman Perhaps we should choose the value directly even higher, because depending on the environment, 2 seconds could also be too little.